zerver/openapi/zulip.yaml

```diff
@@ -11746,6 +11746,194 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /invites:
+    get:
+      operationId: get-invites
+      summary: Get all invites
+      tags: ["invites"]
+      description: |
+        Fetch all the invitations (i.e. email invitations and reusable invitation
+        links) that can be controlled by the user.
+
+        This isn't necessarily the same as all the invitations generated by the
+        user, as administrators can also control invitations that they did not
+        themselves create.
+
+        **Changes**: Prior to Zulip 8.0 (feature level 209), non-admin users could
+        only create email invitations, and therefore the response did not include
+        reusable invitation links for these users.
```

Comment:
"therefore the response did not include" => "therefore the response would never include" might be clearer.